internal/cli/input/secret.go

```diff
@@ -78,7 +78,7 @@ func GetAmazonCredentialsRegion(defaultRegion string) (string, string, map[strin
 }
 
 // GetAmazonCredentials extracts the local credentials from env vars and user profile
-func GetAmazonCredentials() (string, map[string]string, error) {
+func GetAmazonCredentials(createSecret bool) (string, map[string]string, error) {
```

Comment:
I'd call something more general, like `allowSessionTokens` (with opposite logical value in this case).